### DISCUSSION TITLE
Pasting directly into terminal does not work

### DISCUSSION THREAD

**Discussion by raivivek** [wezterm/wezterm#2203] [Q&A] [answered]
Hi -- thanks for Wezterm. I am having an issue with running commands by pasting them directly into the terminal. If I copy anything, a command for example, and paste it into terminal and press "Enter" (without changing anything), the command is not run. But if I delete a character and retype it or just move the cursor bar and then press "enter", the command is executed. Please see an example below:

- First line is "paste" + "Enter"
- Second line is same as above
- Third line is move cursor bar a single character to left and then press enter

![image](https://user-images.githubusercontent.com/3218617/176545251-19e03d1b-d69f-4cb7-bf12-04ea70007518.png)

What seems to be the issue? Thanks!

I'm using `starship` with ZSH shell on a remote server.

**Comment by wez** [comment, marked as answer]
Please open an issue and include the information requested by the form: https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&template=bug.yml

**Reply by raivivek** [reply]
Okay, [done](https://github.com/wez/wezterm/issues/2204).
